tools/marvin/marvin/cloudstackConnection.py

```diff
@@ -143,7 +143,7 @@ def __sign(self, payload):
             ["=".join(
                 [str.lower(r[0]),
                  str.lower(
-                     urllib.quote_plus(str(r[1]))
+                     urllib.quote_plus(str(r[1]), safe="*")
```

Comment:
This is ported from cloudmonkey -- https://github.com/apache/cloudstack-cloudmonkey/pull/11/files
When tests send * in the cmd arg, test failures happened due to how url encoding differs across ACS specific Java code and Python